### PR TITLE
[Issue #5][UI] courses

### LINKED ISSUE #5: issue 3
image
image
image
image

### PR DESCRIPTION
**Pull request by hapo-lamnq**
close #5

### CODE REVIEW

**Comment by son19999** [comment]
LGTM

File: scss/_courses.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
+.background-grey {
+  width: 100%;
+  padding-bottom: 16%;
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: set height theo px

File: scss/_courses.scss [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  background-image: linear-gradient(0deg, #e5e5e5, white);
+}
+
+.courses {
+  width: 100%;
+  height: auto;
+  margin-top: -10%;
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: set px

File: scss/_courses.scss [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+      display: flex;
+      justify-content: center;
+    }
+  }
+}
+
+.card {
+  border-radius: 15px;
+  box-shadow: $box-shadow;
+  width: 75%;
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: width 100%

**Comment by son19999** [comment]
LGTM

**Comment by son19999** [comment]
LGTM

**Comment by son19999** [comment]
LGTM

**Comment by hapo-sonnx** [comment]
LGTM

File: index.html [hunk 4]
```diff
@@ -9,45 +9,50 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (hapo-nghialuu) [inline index.html]: đưa script này xuống dưới
hỏi thêm Khánh là tại sao cần đưa script xuống dưới nhé

File: index.html [hunk 5]
```diff
@@ -9,45 +9,50 @@
...
-                        </li>
-                        <li class="nav-item active">
-                            <a class="nav-link text-secondary" href="#">Login/Register</a>
-                        </li>
-                        <li class="nav-item active">
-                            <a class="nav-link text-secondary" href="#">Profile</a>
-                        </li>
-                    </ul>
+            <div class="row row-nav">
+                <div class="col-lg-4 logo" style="background-color: none;">
```
Comment (hapo-nghialuu) [inline index.html]: style inline là không được dùng nhé

File: index.html [hunk 6]
```diff
@@ -9,45 +9,50 @@
...
+                    <a class="navbar-brand" href="#">
+                        <img src="./images/hapo_learn.png" class="d-inline-block align-top img-logo"
+                            alt="logo-hapo-learn">
+                    </a>
+                </div>
+                <div class="col-lg-8 btn-navbar" style="background-color: none;">
```
Comment (hapo-nghialuu) [inline index.html]: vẫn style inline

File: scss/_courses.scss [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+  .img-php {
+    width: 90%;
+    padding: 21px;
+  }
+}
+
+.card-body {
+  text-align: center;
+  padding: 5%;
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: padding px đi

File: scss/_courses.scss [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+    margin: 0;
+    font-size: 16px;
+    color: $txt-color;
+    margin-bottom: 0.5rem;
+    @include three-dot;
+  }
+
+  .btn-takeCourse {
+    background: $bg-color;
+    box-shadow: inset 0px 1.5px 2px #6f8710;
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: định nghĩa màu

**Comment by hapo-nghialuu** [comment]
close #5

**Comment by hapo-tungdq** [comment]
LGTM

File: .gitignore [hunk 9]
```diff
@@ -1,3 +1,4 @@
 css
 style.css
 style.css.map
+test.html
```
Comment (hapo-nghialuu) [inline .gitignore]: thêm cái này vào làm gì thế em?

File: scss/_courses.scss [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+  .card-text {
+    font-family: $font-Roboto;
+    margin: 0;
+    font-size: 16px;
+    color: $txt-color;
+    margin-bottom: 0.5rem;
+    @include three-dot;
+  }
+
+  .btn-takeCourse {
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: đặt lại tên btn này nhé, không dùng chữ hoa, tất cả là chữ thường và dấu - mà

File: scss/_courses.scss [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+    .card-title {
+      font-weight: 700;
+      font-size: 14px;
+    }
+
+    .card-text {
+      font-size: 13px;
+    }
+
+    .btn-takeCourse {
```
Comment (hapo-nghialuu) [inline scss/_courses.scss]: tìm và sửa các lỗi tương tự như thế này cho anh nhé

**Comment by hapo-tungdq** [comment]
LGTM

**Review by hapo-nghialuu — APPROVED** [review]

**Comment by hapo-nghialuu** [comment]
LGTM
